spec.html

```diff
@@ -171,7 +171,7 @@ <h1>Hosts and Implementations</h1>
     <p>A <dfn id="host">host</dfn> is an external source that further defines facilities listed in Annex <emu-xref href="#sec-host-layering-points"></emu-xref> but does not further define other implementation-defined or implementation-approximated facilities. In informal use, a host refers to the set of all implementations, such as the set of all web browsers, that interface with this specification in the same way via Annex <emu-xref href="#sec-host-layering-points"></emu-xref>. A host is often an external specification, such as WHATWG HTML (<a href="https://html.spec.whatwg.org/">https://html.spec.whatwg.org/</a>). In other words, facilities that are host-defined are often further defined in external specifications.</p>
     <p>A <dfn id="host-hook">host hook</dfn> is an abstract operation that is defined in whole or in part by an external source. All host hooks must be listed in Annex <emu-xref href="#sec-host-layering-points"></emu-xref>. A host hook must conform to at least the following requirements:</p>
     <ul>
-      <li>It must return either a normal completion or a throw completion.</li>
+      <li>It must return a completion with a [[Type]] value of ~normal~ or ~throw~.</li>
```

Comment:
Now that you've defined the shorthands, you don't need to make this change.

Reply:
I made it to be consistent with #2448 as you pointed out. Any preference on going with this or staying with "normal or throw completion" for both?

Reply:
"normal completion or throw completion" for both seems best to me. It means your addition of the definitions of those terms will race. Maybe just split it out and we get that in now?

Reply:
Let's just sequence this one before the other -- meeting's in 2 days and we should get consensus for the other one by then.